src/main/java/io/cdap/delta/bigquery/BigQueryAssessor.java

```diff
@@ -98,8 +98,12 @@ public TableAssessment assess(StandardizedTableDetail tableDetail) {
         new Problem("Missing Primary Key",
                     String.format("Table '%s' in database '%s' must have a primary key in order to be replicated",
                                   tableName, dbName),
-                    "Please alter the table to use a primary key, or select a different table",
-                    "Not able to replicate this table to BigQuery"));
+                    "Please set the property 'source.connector.custom.primary.key' = " +
+                      "`<fully-qualified table>:<a comma-separated list of columns>' in the preferences to columns " +
+                      "that can be used as an indexes. An index column is required to create a primary key in BigQuery " +
```

Comment:
need to fix the line length , it's exceeding 120 characters

Reply:
or cherry-pick https://github.com/data-integrations/bigquery-delta-plugins/pull/169 together